tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
+expertsystem==0.1.2a0
```

Comment:
Why is this included here. Either we include all of the requirements or that comes from the other list

Reply:
Ah yeah this is a matter we need to discuss. Because by now we have one under the requirements for documentation (namely to run the notebooks):
https://github.com/ComPWA/tensorwaves/blob/2b85e3cd6fe4bab07b7dd958112b6e56f97f15bb/doc/requirements.txt#L2

and one under `tests` (in able to run `pytest`):
https://github.com/ComPWA/tensorwaves/blob/2b85e3cd6fe4bab07b7dd958112b6e56f97f15bb/tests/requirements.txt#L1

I did not put the expertsystem under the official requirements, because the `requirements.txt` of the `tensorwaves` package itself, because it's not a requirement for the source code itself.

For me it again show how the whole reliance on `dict` only creates the illusion that `tensorwaves` is decoupled from the `expertsystem`: in practice there is a tight coupling to the version of the `expertsystem` and it would be better to implement that as a strict dependency (even better to use recipe functionality from the expertsystem itself instead of `dict`s).

Reply:
That's true. So I guess the interface between the expert system and tensorwaves should be a graph structure instead of that dict...